swagger_parser/lib/src/parser/parser.dart

```diff
@@ -62,6 +62,8 @@ class OpenApiJsonParser {
   static const _schemasVar = 'schemas';
   static const _pathsVar = 'paths';
   static const _allOfVar = 'allOf';
+  static const _oneOfVar = 'oneOf';
```

Comment:
I think it doesn't make sense to add variables, because they are not used anywhere yet. This can be added when the feature with anyOf and one Of is already implemented.

Reply:
Removed, sorry, I didn't notice them in the commit.